### PR TITLE
Update train.py with parameters_names as required by optimizer initialization

### PR DESCRIPTION
**Pull request by armusc**

### CODE REVIEW

File: egs/librispeech/ASR/pruned_transducer_stateless7_ctc/train.py [hunk 1]
```diff
@@ -1035,8 +1035,12 @@ def run(rank, world_size, args):
     if world_size > 1:
         logging.info("Using DDP")
         model = DDP(model, device_ids=[rank], find_unused_parameters=True)
-
-    optimizer = ScaledAdam(model.parameters(), lr=params.base_lr, clipping_scale=2.0)
+        
+    parameters_names = []
```
Comment (csukuangfj) [inline egs/librispeech/ASR/pruned_transducer_stateless7_ctc/train.py:1039]: Wondering why it did not throw an assertion error before.

@yaozengwei Could you please have a check?

Comment (yaozengwei) [reply egs/librispeech/ASR/pruned_transducer_stateless7_ctc/train.py:1039]: I see. I trained this model before the optimizer files changed (see 4fee3e7f1ea6c2aefe7594e325ede1e530e54d3d).

**Comment by desh2608** [comment]
Could you also please make the same change here: https://github.com/k2-fsa/icefall/blob/0e325c8782c8b9178cf0f2b030e49ae64f2b091d/egs/ami/ASR/pruned_transducer_stateless7/train.py#L975

**Comment by armusc** [comment]
I formatted the file with black, not sure what issue is pointing out

**Comment by csukuangfj** [comment]
> I formatted the file with black, not sure what issue is pointing out

Which version of black are you using?

It expects:
https://github.com/k2-fsa/icefall/blob/0e325c8782c8b9178cf0f2b030e49ae64f2b091d/.github/workflows/style_check.yml#L52

https://github.com/k2-fsa/icefall/blob/0e325c8782c8b9178cf0f2b030e49ae64f2b091d/.pre-commit-config.yaml#L2-L3

**Comment by csukuangfj** [comment]
Here is the error reported by the CI:
https://github.com/k2-fsa/icefall/actions/runs/3646630949/jobs/6158121816#step:6:1

Screen Shot 2022-12-08 at 5 20 50 PM

**Comment by armusc** [comment]
I use indeed black 22.3.0
when the arguments of ScaledAdam were on the same line, running "black" on the file would reformat it the way it is now

**Comment by csukuangfj** [comment]
I just tried to run `black` on your changes, locally:

Screen Shot 2022-12-08 at 5 45 43 PM

So it matches the style issue reported by the CI.

**Comment by csukuangfj** [comment]
I see the reason.

Screen Shot 2022-12-08 at 5 47 40 PM

You put extra blanks on line 1038.

**Comment by csukuangfj** [comment]
> Could you also please make the same change here:
> 
> https://github.com/k2-fsa/icefall/blob/0e325c8782c8b9178cf0f2b030e49ae64f2b091d/egs/ami/ASR/pruned_transducer_stateless7/train.py#L975

@armusc Would you mind fixing this comment?

**Comment by yfyeung** [comment]
fix #741

**Pull request merged**